### DISCUSSION TITLE
hexo d 失败

### DISCUSSION THREAD

**Discussion by Disatace** [hexojs/hexo#5083] [Question / Answer] [answered]
## Check List

Please check followings before submitting a new issue.

- [ ] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [ ] I have already searched existing issues and they are not help  to me
- [ ] I examined error or warning messages and it's difficult to solve
- [ ] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [ ] Node.js is higher than [minimum required version](https://hexo.io/docs/#Minimum-required-Node-js-version)

## How to reproduce?

* Step1
* Step2
* etc...

## Environment & Settings

**Node.js & npm version(`node -v && npm -v`)**

```
```

**Your site `_config.yml`** (Optional)

```yaml
```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
```

**Your package.json `package.json`**

```
```

**Comment by Disatace** [comment]
原本将腾讯云服务器作为远程仓库
现在换其他服务器
hexo clean g s都是正常
hexo d失败显示
Branch 'master' set up to track remote branch 'master' from 'git@124.71.184.xxx:/home/git/repos/tailblog.git'.
git clone 失败
Cloning into 'tailblog'...
fatal: protocol 'git@124.71.184.136' is not supported
ssh连接正常

**Reply by stevenjoezhang** [reply]
把 `xxx:/` 中间的冒号删掉

**Comment by Disatace** [comment, marked as answer]
已解决 冒号后斜杠多了一个 并且域名忘记解析到新服务器了
